src/lib/Cards/GameCardHighlighted.svelte

```diff
@@ -1,57 +1,40 @@
-<section>
-	<h1>Uitgelichte spellen</h1>
-	<ul>
+<ul>
+	<li>
 		<a href="spellen/#">
-			<div>
+			<article>
```

Comment:
Komen misschien nog wat conflicten met die van mij nu. Heb in principe niet aan die `div` gezeten bij mij, maar gaan het zien.

Staat overigens nu wel dubbel in `Changes`. Ik weet niet of we straks opeens `Cards/` & `cards/` hebben. Hier wel ff goed op letten, voordat we mergen. Is namelijk bij meer files/folders.

Reply:
dit hebben we gefixt in deze branch, maar merge :/